server/libs/infra/src/repositories/media.repository.ts

```diff
@@ -22,11 +28,31 @@ export class MediaRepository implements IMediaRepository {
         return;
 
       case 'jpeg':
-        await sharp(input, { failOnError: false })
-          .resize(options.size, options.size, { fit: 'outside', withoutEnlargement: true })
-          .jpeg()
-          .rotate()
-          .toFile(output);
+        if (!options.raw) {
```

Comment:
It may be possible to remove a level of indentation by returning early. Right now it looks like:

```ts
if (raw) {

} else {

}
```

It could look like:

```
if (raw) {
    return
}

// not raw code
```

Might be nice.

Reply:
Yeah, that's a good catch. I usually prefer code like that.